### PR TITLE
Tweaked gifwit library template

### PR DESCRIPTION
**Pull request by streeter**

### CODE REVIEW

**Comment by jglovier** [comment]
ZOMG thank you for addressing this!! :heart: :heart: :heart:

File: script/build_site_index [hunk 1]
```diff
@@ -42,7 +42,7 @@ list.ul do |builder|
     builder.li do |dir_element|
       dir_element.text!(dir)
       dir_element.ul do |ul|
-        Dir.glob("#{dir}/*.gif") do |gif|
+        Dir.glob("#{dir}/**/*.gif") do |gif|
```
Comment (jglovier) [inline script/build_site_index]: What specifically does this do?

Comment (streeter) [inline script/build_site_index]: Ah, right. So it causes [`glob` to search directories recursively](http://ruby-doc.org/core-1.9.3/Dir.html#method-c-glob). It also searches the directory `"#{dir}/./*.gif"`, or, `"#{dir}/*.gif"`. So this allows for you to have a directory structure like [`politics/potus/*.gifs`](https://github.com/streeter/gifs/tree/gh-pages/politics/potus), and then tag the gif with each path component. `politics/potus/obama-kickdoor.gif` becomes `["politics", "potus", "obama", "kickdoor"]`

**Comment by streeter** [comment]
I'll rebase this PR, so that it can merge cleanly now.

**Comment by streeter** [comment]
Rebased the branch, so should be good to merge now.

File: script/build_site_index (hunk 1 above)
Comment (jglovier) [inline script/build_site_index]: :metal:

**Comment by jglovier** [comment]
:zap: :tada:

**Pull request merged**